istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -141,7 +143,7 @@ spec:
             value: "{{ .Values.pilot.enableProtocolSniffingForInbound }}"
 {{- if .Values.global.istiod.enabled }}
           - name: WEBHOOK
-            value: istiod
+            value: istio-sidecar-injector
```

Comment:
Just curious: why do you make this change?

Reply:
There is no webhook named "istiod". The webhook name is istio-sidecar-injector. see https://github.com/istio/istio/issues/19780